monorepo-folders/packages/backend-apis/temporal-client.ts

```diff
@@ -13,13 +13,13 @@ export async function runWorkflow() {
   // Invoke the `WorkflowA` Workflow, only resolved when the workflow completes
   const result = await client.execute(WorkflowA, {
     taskQueue: 'tutorial',
-    workflowId: 'workflow-a',
+    workflowId: 'workflow-a-' + Date.now(),
```

Comment:
Why did we add the date here?

Reply:
Worth @sw-yx 's thoughts here. My issue with this as implemented was that I couldn't run the workflow twice from the UI, the 2nd time I'd get "ALREADY_EXISTS: Workflow execution is already running. WorkflowId: workflow-a". Which is a bit of friction.

What do you think?

Reply:
It shouldn't return ALREADY_EXISTS if the workflow finishes.
I'd at least add a comment that users should provide a meaningful business ID.

Reply:
```suggestion
    workflowId: 'workflow-a-' + Date.now(), // TODO: remember to replace this with a meaningful business ID
```